src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import styles from './CourseInfo.module.css';
+
+import Button from '../../common/Button/Button';
+import { mockCourses } from '../../data/data';
+import getCoursesDuration from '../../helpers/getCoursesDuration';
+import getAuthorNames from '../../helpers/getAuthorNames';
+
+const CourseInfo = () => {
+  let { courseId } = useParams();
+  let title;
+  let description;
+  let duration;
+  let authors;
+  let creationDate;
+  mockCourses.forEach((element) => {
+    if (courseId == element.id) {
+      title = element.title;
+      description = element.description;
+      duration = element.duration;
+      authors = element.authors;
+      creationDate = element.creationDate;
+    }
+  });
+
+  return (
+    <div className={styles.courseInfoForm}>
+      <div className={styles.backToCoursesLink}>
+        <Link to={'/courses'}>
+          <Button buttonText={' \u1438 Back to courses'} />
+        </Link>
+      </div>
+      <h1 className={styles.title}>{title}</h1>
+      <div className={styles.formBody}>
+        <div className={styles.description}>{description}</div>
+        <div className={styles.coursesForm}>
+          <div>
+            <h1 className={styles.properties}>ID: </h1>
```

Comment:
Let's use `p` tag, please, to avoid HTML semantic rules violation for describing entities